runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -4194,3 +4194,10 @@ J9::Z::CodeGenerator::supportsTrapsInTMRegion()
    {
    return self()->comp()->target().isZOS();
    }
+
+bool J9::Z::CodeGenerator::supportsInliningOfIsAssignableFrom()
+   {
+   return false; // will be taken out once new inlined tests are added
+   static const bool disableInliningOfIsAssignableFrom = feGetEnv("TR_DisableInliningOfIsAssignableFrom") != NULL;
```

Comment:
Why do you need this ? Parent Codegenerator class implementation will return false. As you are not yet enabling this, We should take this override out of this PR.